### ISSUE TITLE
expend the output of CNN

### ISSUE THREAD

**Issue by chamas14** [tensorflow/tensorflow#4655]
I am trying to expend the outputs of my network from 11 to 12 outputs, i have restored the previous checkpoint that is already retrained in 11 outputs. I got the answer from  here http://stackoverflow.com/questions/34913762/how-to-expand-a-tensorflow-variable in This question told me how to change the shape of the variable, to expand it to fit another row of weights, but I don't know if i initialize the weight and biases in the right way. Actually i don't have error but the the test accuracy is decreased from 95% to 9%. may be there is somewhere wrong issue in the code. that's the code:

     w_b_not = {
      'weight_4': tf.Variable(tf.random_normal([num_hidden, num_labels], stddev=0.1)),
      'bias_4'  : tf.Variable(tf.constant(1.0, shape=[num_labels])),}

     w_b = {
      'wc1_0': tf.Variable(tf.random_normal([patch_size_1, patch_size_1, num_channels, depth],stddev=0.1)),
       .....
      'bc1_0' : tf.Variable(tf.zeros([depth]))}

     .... #here is the networks model

     num_steps = 1001 
     with tf.Session(graph=graph) as sess:
        ckpt = ('path_of_checkpoint.ckpt')
        if os.path.isfile(ckpt) :
           layer6_weights = tf.Variable(tf.random_normal([num_hidden, num_labels], stddev=0.1))
           layer6_biases = tf.Variable(tf.constant(1.0, shape=[num_labels]))

      n_w_b = {
      'new_layer_weights' : tf.concat(0,[w_b_not['weight_4'], layer6_weights]),
      'new_layer_biases' : tf.concat(0,[w_b_not['bias_4'], layer6_biases])}
      resize_var_1 = tf.assign(w_b_not['weight_4'], n_w_b['new_layer_weights'], validate_shape=False)
      resize_var_2 = tf.assign(w_b_not['bias_4'], n_w_b['new_layer_biases'], validate_shape=False)
      logits = tf.get_collection('logits')[0]
      w_b_new_saver = tf.train.Saver()
      init_op = tf.initialize_all_variables()        
      w_b_saver.restore(sess, ckpt)
      print("restore complete")
      for step in xrange(num_steps):
        sess.run(init_op)
      print('Test accuracy: %.1f%%' % accuracy(test_prediction.eval() , test_labels,  force = False ))
